utils.js

```diff
@@ -44,3 +29,27 @@ function filterOas(files) {
   });
   return oas;
 }
+
+module.exports = {
+  listOas(globs) {
+    if (
+      fs.existsSync(path.join(process.cwd(), 'api.config.json')) &&
+      fs.existsSync(path.join(process.cwd(), 'endpoints'))
+    ) {
+      return ['api.config.json'];
+    }
+
+    if (!globs || !globs.length) {
+      // eslint-disable-next-line no-param-reassign
+      globs = ['**/*.{yaml,yml,json}'];
+    }
+
+    let out = [];
+    globs.forEach(g => {
+      // Concat them and filter dupes
+      const list = filterOas(glob.sync(g));
```

Comment:
does 'glob.sync' use 'process.cwd()'? I assume it does. if it doesn't it should.